installers/rpm-code/codes/hypre.sh

```diff
@@ -2,9 +2,25 @@
 
 hypre_main() {
     codes_dependencies common
-    codes_download https://github.com/hypre-space/hypre/archive/v2.11.2.tar.gz
+    codes_download https://github.com/hypre-space/hypre/archive/v2.24.0.tar.gz
     cd src
-    ./configure --prefix="${codes_dir[prefix]}"
+    local a=''
+    if [[ ${1:-} == 'gpu-only' ]]; then
+        #POSIT: container-jupyter-nvidia (cuda-11.1)
+        # and running on Tesla V100 (arch 70)
+        a=(
+            --with-mpi
+            --enable-unified-memory
+            --prefix="${codes_dir[prefix]}"
+            --with-MPI
+            --with-cuda
+            --with-cuda-home='/usr/local/cuda-11.1'
```

Comment:
You don't need quotes. Would be nice to get this from somewhere, but I know that's hard.

Reply:
495c144